pkg/kf/internal/envutil/envutil_test.go

```diff
@@ -203,10 +203,9 @@ func ExampleGetServiceEnvVars() {
 
 	// Output: Key FOO Value 2
 	// Key BAR Value 0
-}
 
-func ExampleGetServiceEnvVars_EmptyService() {
```

Comment:
This could also be `ExampleGetServiceEnvVars_emptyService` IIRC

Reply:
Ah nice!